src/DbConnection/MySQLConnUtils.java

```diff
@@ -0,0 +1,28 @@
+package DbConnection;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+  
+public class MySQLConnUtils {
+  
+ public static Connection getMySQLConnection()
+         throws ClassNotFoundException, SQLException {
+     String hostName = "localhost";
```

Comment:
These parameters should not be hardcoded in the code. They should be saved in a properties file.